doc/API.md

```diff
@@ -68,27 +69,37 @@ The currently-defined error responses are:
 * status code 400, errno 101: Invalid email. Missing or malformed email.
 * status code 400, errno 102: Invalid password. The password should have a minimum of 8 chars.
 * status code 400, errno 103: Missing or malformed authentication header.
+* status code 400, errno 104: Invalid user id.
 * status code 401, errno 401: Unauthorized. If credentials are not valid.
 * status code 409, errno 409: Conflict. The user is already registered.
 * status code 410, errno 410: Gone. The resource is no more available. Don't insist.
+* status code 423, errno 423: Locked. You are trying to delete the last user with admin privileges. That's forbidden.
 * status code 501, errno 501: Internal server error.
 * any status code, errno 999: Unknown error
 
 # API Endpoints
 
 * Setup
     * [POST /setup](#post-setup)
-    * [POST /login](#post-login)
+* Login
+    * [POST /login](#post-login) :lock: (CORS allowed)
+* User management
+    * [POST /users](#post-users) :lock:
+    * [GET /users](#get-users) :lock:
+    * [GET /users/:id](#get-usersid) :lock:
+    * [PUT /users/:id](#put-usersid) :lock:
+    * [PUT /users/:id/activate](#put-usersidactivate)
+    * [DELETE /users/:id](#delete-usersid) :lock:
 
 ## POST /setup
-Allow to initiate the box by registering an admin user. CORS is not allowed for this endpoint.
+Allow to initiate the box by registering an admin user.
 ### Request
 ___Parameters___
 * email - Admin email.
-* username - User name. Defaults to "admin".
-* password - Admin password.
+* username - Optional. User name. Defaults to "admin".
```

Comment:
I think that I commented before, but since I've been doing a weird review checking commit by commit, now it doesn't appear.

Anyway, do we need an optional user name? If we are going to move towards email as an identifier, I think we just can skip the username.